aspnetcore/blazor/hosting-models.md

```diff
@@ -150,6 +80,21 @@ Blazor WebAssembly includes support for trimming unused code from .NET Core fram
 
 ::: moniker-end
 
+## Hosting model selection
+
+Choice of the Blazor hosting model is an early consideration for Blazor app development. The following table shows the primary considerations for selecting the hosting model.
+
+| &nbsp; | Blazor Server | Blazor WebAssembly |
+| --- | :---: | :---: |
+| Wide .NET Core API compatibility              | ✔️ | ❌ |
+| Wide 3rd party .NET tooling support           | ✔️ | ❌ |
+| Small payload size with fast initial download | ✔️ | ❌ |
+| App code secure and private on the server     | ✔️ | ❌&dagger; |
+| Serverless deployment                         | ❌ | ✔️ |
```

Comment:
I'm intentionally trying to look at this from a newbie's perspective and `Serverless` may be a word they're not familiar with. What if we use a simpler description here:

```suggestion
| Possibility to run apps without a backing server                        | ❌ | ✔️ |
```

Note, that this is just an idea. I will be happy to get a push back on this.